package/yast2-packager.spec

```diff
@@ -26,24 +26,28 @@ Source0:        %{name}-%{version}.tar.bz2
 Url:            https://github.com/yast/yast-packager
 BuildRequires:  update-desktop-files
 BuildRequires:  yast2-devtools >= 3.1.10
+BuildRequires:  rubygem(%{rb_default_ruby_abi}:cfa) >= 0.5.0
 BuildRequires:  rubygem(%{rb_default_ruby_abi}:rspec)
 BuildRequires:  rubygem(%{rb_default_ruby_abi}:yast-rake)
-BuildRequires:  rubygem(%{rb_default_ruby_abi}:cfa) >= 0.5.0
 
 # Y2Storage::Device#exists_in_raw_probed
 BuildRequires:  yast2-storage-ng >= 4.0.141
+# breake the yast2-packager -> yast2-storage-ng -> yast2-packager build cycle
+#!BuildIgnore: yast2-packager
+
+BuildRequires:  yast2-transfer
```

Comment:
this can be easily avoided by module stub of HTTP and FTP. I did it when I try to break this cycle. So not needed.

Reply:
Thanks, replaced by stubbing the modules.